algorithms/kernel/stump/stump_classification_train_aux.i

```diff
@@ -51,79 +52,82 @@ namespace internal
 {
 
 template <Method method, typename algorithmFPtype, CpuType cpu>
-services::Status StumpTrainKernel<method, algorithmFPtype, cpu>::changeMinusOneToZero(NumericTable *yTable)
+Status StumpTrainKernel<method, algorithmFPtype, cpu>::changeMinusOneToZero(const algorithmFPtype *yArray, algorithmFPtype *yZeroOne, size_t nVectors)
 {
-    services::Status s;
-    const size_t nVectors  = yTable->getNumberOfRows();
-    WriteColumns<algorithmFPtype, cpu> y(const_cast<NumericTable *>(yTable), 0, 0, nVectors);
-    DAAL_CHECK_STATUS(s, y.status());
-    algorithmFPtype *yArray = y.get();
+    const algorithmFPtype one = 1.0;
+    const algorithmFPtype zero = 0.0;
+    const algorithmFPtype minusOne = -1.0;
     for(size_t i = 0; i < nVectors; i++)
     {
-        if(yArray[i] == -1) { yArray[i] = 0; }
+        if(yArray[i] == minusOne) { yZeroOne[i] = zero; }
```

Comment:
Add comment about usage of == for fp type

Reply:
ok